loop-de-loop.ipynb

```diff
@@ -54,7 +54,12 @@
    "cell_type": "markdown",
```

Comment:
<p>The loop should iterate each row, instead of each column.</p><p>The printed output for each iteration should be like:</p><p>| 0 | [1,2,3,4] | 1 | 1 | [1] |</p><p>| 1 | [11,12,13,14] | 12 | 12 | [1,12] |</p><p>| 2 | [21,22,23,24] | 23 | 23 | [1,12,23] |</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UMD-ENMA165/loop-de-loop/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>